modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
umm good point, because I believe the only thing using this export
```Javascript
export { OSMNode as Node } from './node';
```
is rollup which will eventually make this function `iD.Node`, so I had to re-export it this way.
Although, I don't know the best solution. It feels like a hack. 
what do you suggest ?